dir.props

```diff
@@ -53,10 +53,10 @@
 
     <!-- Packaging configuration -->
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)pkg/Microsoft.NETCore.Platforms/runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(PackagesDir)Microsoft.NETCore.Platforms/1.0.1-rc2-23712/runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
I kind of worry about keeping this version number updated correctly. @ericstj any thoughts on how to get this information in another way?